config/phenomenal_tools2container.yaml

```diff
@@ -272,7 +272,7 @@ assignment:
 - docker_image_override: batman
   docker_owner_override: phnmnl
   docker_repo_override: container-registry.phenomenal-h2020.eu
-  docker_tag_override: v1.2.12.0_cv1.0.65
+  docker_tag_override: dev_v1.2.12.0_cv1.0.69
```

Comment:
Don't change this to dev please, at least not in the release branch

Reply:
Do I have to change the version from 65 to 69?

Reply:
....mmm... I see, there was no release made for batman.... I'll try to do a release of batman tomorrow so that we can put something adequate here...

Reply:
ignore this @ilveroluca